PSKoans/ExportedTypes.ps1

```diff
@@ -11,4 +25,33 @@ class KoanAttribute : System.Attribute {
     KoanAttribute() {
         $this.Position = [uint32]::MaxValue
     }
+
+    [void] InvokeHelpInfo() {
+        if ($this.HelpAction) {
+            $this.InvokeHelpAction()
+        }
+        elseif ($this.HelpPath) {
+            $this.InvokeHelpPath()
+        }
+        elseif ($this.HelpURL) {
+            $this.InvokeHelpURL()
+        }
+    }
+
+    [void] InvokeHelpAction() {
+        $this.HelpAction.InvokeReturnAsIs()
+    }
+
+    [void] InvokeHelpURL() {
+        Write-ConsoleLine "Confusion is only the beginning"
+        Write-ConsoleLine "Your defeat has only made you stronger"
+        Write-ConsoleLine ("Learn through Zen: {0}" -f $this.HelpURL)
+    }
+
+    [void] InvokeHelpPath() {
+        Write-ConsoleLine "Confusion is only the beginning"
+        Write-ConsoleLine "To become Master must be able to snatch the coin from my hand"
```

Comment:
Might be a _little_ forced here, haha. 😄 

Leave it out for now and we can add something more rooted a bit later if we need to.